form.yml

```diff
@@ -3,7 +3,15 @@ cluster: "pitzer-login"
 form:
   - bc_num_hours
   - working_dir
+  - version
 attributes:
+  version:
+    widget: "select"
+    label: "Codeserver Version"
+    options:
+      - 3.9.3
+      - 3.4.1
```

Comment:
Can these be 

```
[ "3.9", "3.9.1" ]
```

instead?  it may seem like a nitpick, but it gives us the opportunity to add bugfixes without giving users yet another choice.

Reply:
Yeah no problem. I need to fix the ghost user commit anyway so this actually makes that easier.